frontend/src/components/pages/search/Search.tsx

```diff
@@ -106,8 +106,6 @@ export const SearchUI: React.FC = () => {
                   <FormattedMessage id="search.orderedByRelevance" />
                 </RankingInfo>
 
-                <Separator className="w-full mt-6 desktop:block hidden" />
```

Comment:
Attention tu as oublié de remplacer le séparateur horizontal
![image](https://user-images.githubusercontent.com/48312749/105853366-e770a200-5fe5-11eb-800a-ffc34c0cd635.png)

Reply:
En fait non, une fois que la PR sera mergée il réaparaitra - j'ai créé un composant "Separator" que j'ai utilisé sur la page détail, du coup je l'ai pushé sur ma branche de dev detail et pas sur la branche de dev style

Reply:
C'est noté pour l'autre regression

Reply:
Ohh okay ça marche top du coup :)